README.md

```diff
@@ -54,3 +54,58 @@ spark.optimizeSpatialAuto(sourceTable, outputTable, outputLocation)
 // optimize with the user defined block size
 spark.optimizeSpatial(sourceTable, outputTable, outputLocation, blockSize = 20097000)
 ```
+
+### Registering Optimizations on Databricks
+
+> This section and approach are based on [docs]](https://sedona.apache.org/setup/databricks/)
+> from Apache Sedona.
+
+Installing and registering spark optimizations can be tricky. As is often the case when
+working with distributed systems, there are some fundamental sequencing issues that are
+important to understand. Roughly, the startup of a databricks cluster looks something like
+this:
+
+1. The JVM process starts with the cluster default classpath
+2. The spark config is initialized (Here's where we want to enable optimizations)
+3. VFS/DBFS user class paths are mounted
+
+The jar which contains classes that are referenced in step 2 isn't available prior to step 3!
+Fortunately, it is possible to set up initialization scripts which run prior to step 1 and
+which we can use on databricks to ensure our classes are available by the time a cluster
+loads its spark config.
+
+First, write a script to DBFS which can be used to copy jars from DBFS to the local filesystem
+on master. (Note that you will replace 'analytics-toolbox' with whatever your jar is named. For
+example, this is the name of a SNAPSHOT jar used to test these optimizations:
+core-assembly-0.0.12+1-29cd9e81+20220412-0752-SNAPSHOT.jar.) This script can be written using
+a notebook cell:
+```bash
+%sh 
+
+# Create init script directory for Carto
+mkdir -p /dbfs/FileStore/carto/
+
+# Create init script
+cat > /dbfs/FileStore/carto/carto-init.sh <<'EOF'
+#!/bin/bash
+#
+# 
+# On cluster startup, this script will copy the Carto jars to the cluster's default jar directory.
+# In order to activate Carto ST_Intersection plan optimization: "com.carto.analyticstoolbox.spark.rules.SpatialFilterPushdownRules"
+
+# cp /dbfs/FileStore/jars/maven/com/carto/analyticstoolbox/*.jar /databricks/jars
+# tmp solution to handle the assembly jar
+cp /dbfs/FileStore/jars/analytics-toolbox.jar /databricks/jars
```

Comment:
Lets make it easier for users, we don't need this line probably, that's also a good question how to get `/dbfs/FileStore/jars/analytics-toolbox.jar` uploaded.

Reply:
Is that not outside the scope of this since we're assuming some familiarity with databricks? I'm hesitant to post UI screenshots given how rough some of the UI is and how likely to change in the future